source/Halibut.CertificateGenerator/CertificateGenerator.cs

```diff
@@ -78,7 +78,7 @@ static X509Certificate2 AttemptToGenerate(string fullName)
                    };
         }
 
-#if !__MonoCS__
+#if NETFRAMEWORK
```

Comment:
Couldn't see an xplat alternative to `CryptoKeySecurity` - there may be more work todo here.